private/reducer.rkt

```diff
@@ -1,11 +1,11 @@
-#lang racket/base
+#lang parendown racket/base
 
 (require racket/contract/base)
 
 (provide
  for/reducer
- for*/reducer
- (contract-out
+ for*/reducer)
+(provide #/contract-out
```

Comment:
Why two `provide` forms instead of a single combined one?

Reply:
I was changing the other files to use `(provide #/contract-out ...)` to demonstrate that Parendown can reduce nesting of `provide` forms. In this case, only a certain part of this `provide` form used `contract-out`, but I figured I'd better use Parendown here too or it wouldn't be consistent with the other files.

This is something I do in my own code. I'll `(provide #/contract-out ...)` the functions in a module, but merely `(provide ...)` the macros since they can't be given contracts this way. I don't know if there's any reason to *like* doing it this way, but it seemed like something we could talk about.

I dunno. I tend to use multiple provide forms anyway. I have thought sometimes that a single provide form can be preferable for sending a clear message that this is *all* the stuff a module exports, but I also like to break things up into multiple `provide` forms so that I can document them as a group, comment them all out at once, and stuff like that. Breaking them up with whitespace is just as easy to do, so I've got a lot of indecision around this and I'm very open to ideas.

(While we're on the topic of this part of the code, you might notice the `(provide ...)` block is indented by only one space and the `(provide #/contract-out ...)` block is indented by two spaces. I would have adjusted the `(provide #/contract-out ...)` blocks to one space of indentation for conformance with your style, but I didn't want all those changed lines to be distracting in the diff.)